src/parser/warlock/affliction/integrationTests/example.test.ts.snap

```diff
@@ -5241,15 +5340,17 @@ exports[`Affliction Warlock integration test: example log ShadowEmbrace matches
             style={Object {}}
           >
             <small>
-               uptime 
+               
+              uptime 
               <sup>
                 *
               </sup>
             </small>
           </dfn>
           <br />
           0
-           DPS 
+           DPS
+           
           <small>
             0.00
```

Comment:
What's with all the weird additional breaklines here? They shouldn't be there

Reply:
Fixed! Thanks. I think those were inserted by error when converting from `js` to `tsx`.